app/javascript/components/miq-about-modal.jsx

```diff
@@ -0,0 +1,113 @@
+import React from 'react';
+import PropTypes from 'prop-types'
+import { connect } from 'react-redux';
+import { combineReducers, bindActionCreators } from 'redux';
+import { AboutModal } from 'patternfly-react';
+
+const SHOW_ABOUT_MODAL = '@@aboutModal/show';
+const HIDE_ABOUT_MODAL = '@@aboutModal/hide';
+const LOAD_ABOUT_MODAL = '@@aboutModal/load';
```

Comment:
@AparnaKarve we are using prefixes to avoid potential action types collisions.

Reply:
That's actually a good practice and I am going to copy that in my Insights work.

Explanation: As the application gets big and complext, more people work on it. It's likely that 2 devs pick the same name somewhere. Esp. in an app as large as ManageIQ.

It does not matter if you have a collision in the names (left side) because you import them only in some places and you are likely to avoid collision there. (The same names imported from different places will not meet in a single place or you'll notice that.)

However as all the actions types are mixed into a single hash what would happen is this: Multiple reducers would be executed for your action. This will likely cause a bug that might not be seen at first because the correct reducer would be between the ones executed.